db/schema.rb

```diff
@@ -164,6 +164,8 @@
     t.integer "quantity", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "duration_mins"
+    t.integer "duration_days"
```

Comment:
maybe it's too late to have this discussion, but what do you think of having a `quantity_unit` column instead of a quantity field for each unit? maybe it makes things simpler